provisionerd/proto/provisionerd.proto

```diff
@@ -69,6 +69,7 @@ message CompletedJob {
         repeated provisioner.Resource start_resources = 1;
         repeated provisioner.Resource stop_resources = 2;
         repeated provisioner.RichParameter rich_parameters = 3;
+		repeated string git_auth_providers = 4;
```

Comment:
nit: bad indentation. TF formatter didn't catch it?

Reply:
Hmm, I guess not...

Reply:
It was two tabs instead of eight spaces. Weird that it didn't catch...